test/service/test_response.py

```diff
@@ -3498,6 +3498,12 @@ def test_projects_response(self):
         self.assertEqual(expected_bionetwork_name, project['bionetworkName'])
         self.assertTrue(project['isTissueAtlasProject'])
 
+        tissue_atlas = {
+            entry['term']: entry['count']
+            for entry in response_json['termFacets']['tissueAtlas']['terms']
+        }
+        self.assertEqual({'Lung': 1, 'Retina': 1, 'Blood': 1}, tissue_atlas)
+
```

Comment:
I'm surprised that there are no test changes related to `accession` even though we touch the configuration for that field.

Reply:
@hannes-ucsc: "Decided during PL to not add `accession` as a term facet. This means we won't need to cover its aggregation in a test."